UnitTests/GitCommandsTests/GitRevisionTests.cs

```diff
@@ -19,5 +26,31 @@ public void Should_validate_full_sha1_correctly()
             Assert.False(GitRevision.IsFullSha1Hash("0000000000000000000000000000000000000000 "));
             Assert.False(GitRevision.IsFullSha1Hash(" 0000000000000000000000000000000000000000"));
         }
+
+        [TestCase(null)]
+        [TestCase("toto")]
+        [TestCase("toto\ntata\ntiti")]
+        public void Should_have_same_content_When_no_ellipsis(string bodyContent)
+        {
+            var gitRevision = new GitRevision(ObjectId.Random());
+            gitRevision.Body = bodyContent;
+            gitRevision.HasMultiLineMessage = true;
+            Assert.AreEqual(bodyContent, gitRevision.BodySummary);
+        }
+
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        [Test]
+        [TestCase("Too_many_lines")]
+        [TestCase("Too_long_lines")]
+        public void Should_do_ellipsis(string testName)
+        {
+            var content = EmbeddedResourceLoader.Load(Assembly.GetExecutingAssembly(), $"{GetType().Namespace}.MockData.{testName}.bin");
```

Comment:
[nit] this isn't a binary file, but a text file, so .txt is probably more appropriate

Reply:
that's what I made at first but we use `.bin` elsewhere to avoid git manage eol (cf our `.gitattributes` config).

So I was not sure what was the best choice. Even if the tests should works with every line endings...

Reply:
ApprovalTests no longer care about EOL in *.approved.* files.